src/provider/bpmn/BpmnPropertiesProvider.js

```diff
@@ -206,17 +208,23 @@ function getGroups(element) {
 
 export default class BpmnPropertiesProvider {
 
-  constructor(propertiesPanel) {
+  constructor(propertiesPanel, translate) {
     propertiesPanel.registerProvider(this);
+    this._translate = translate;
   }
 
   getGroups(element) {
+    const translate = this._translate;
     return (groups) => {
       groups = groups.concat(getGroups(element));
+      // translate group labels
+      forEach(groups, function(group) {
+        group.label = translate(group.label);
```

Comment:
With this one, we would fix the translation for the `bpmn` properties provider only. We have [multiple ones](https://github.com/bpmn-io/bpmn-js-properties-panel/tree/master/src/provider) for different use cases (Camunda Platform vs. Camunda Cloud specific).

Maybe we should add translation there as well?